pkg/plugins/plugins.go

```diff
@@ -229,7 +241,8 @@ func (scanner *PluginScanner) walker(currentPath string, f os.FileInfo, err erro
 	if f.Name() == "plugin.json" {
 		err := scanner.loadPluginJson(currentPath)
 		if err != nil {
-			log.Error(3, "Plugins: Failed to load plugin json file: %v,  err: %v", currentPath, err)
+			// XXX: Should we use plog here instead of log?
+			log.Error(3, "Plugins: Failed to load plugin %q,  err: %v", filepath.Dir(currentPath), err)
```

Comment:
What is "3"?

Reply:
@kylebrandt It's an argument, called `skip`, to log.Error, which AFAICT actually is unused.

Reply:
I guess it's another indication we might want to switch to using `plog` instead?

Reply:
I elected to standardize the login in pkg/plugin/plugin.go to use the `log` field on PluginManager - didn't see any point in using a mixture of loggers.